rust/ql/src/queries/security/CWE-020/RegexInjection.ql

```diff
@@ -0,0 +1,48 @@
+/**
+ * @name Regular expression injection
+ * @description User input should not be used in regular expressions without first being
+ *              escaped, otherwise a malicious user may be able to inject an expression that
+ *              could modify the meaning of the expression, causing it to match unexpected
+ *              strings.
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 7.8
```

Comment:
I got this score by running the script in the `cwe-scores` repo. I think that's the way to get this number.

Reply:
Yep that's right. :+1: